Add Terraform Module policy

gruebel opened the pull request.

Commit:
[Newton Compute AG: CWP-29368]: Support RedHat non-RPM packages

bruno-motacardoso committed:
Missing A (for Azure) on page title (#1445)

* Missing A on title of page

* Update bc-azr-logging-3.adoc

xumenghan11 committed:
Update add-checkov.adoc (#1414)

typo "prima", corrected to "prisma"

Commit:
[Compute AG]: Language/grammar/image fixes (#1444)

Pubs-MV committed:
[Compute AG] VM Scanning Updates (#1434)

* [Compute AG] VM Scanning Updates

Request from Marc Lawrrence and Vitaly for https://github.com/PaloAltoNetworks/prisma-cloud-docs/issues/1407

* Update vm_image_scanning.adoc

Verified some content updates with Alex

* Update vm_image_scanning.adoc

Pubs-MV committed:
Update vm_image_scanning.adoc (#1449)

fixed syntax

gruebel committed:
Add Terraform Module policy

tsmithv11 commented on code-security/policy-reference/supply-chain-policies/terraform-policies/ensure-terraform-module-sources-use-git-url-with-commit-hash-revision.adoc:
```diff
@@ -0,0 +1,68 @@
+
+Terraform modules are a collection of multiple resource configuration to offer an easy way of repeatable and reusable code logic.
+The most common way is to consume them through the public Terraform registry, which are connected to a VCS, like GitHub.
+These approach is problematic, because the versions are not immutable and can be easily changed, which makes the code vulnerable to a Supply Chain Attack.
```
```suggestion
This approach is problematic, because the module versions are not immutable and the module can be changed without changing the version, which makes the code vulnerable to a Supply Chain Attack.
```

tsmithv11 reviewed (approved):
@gruebel thank you!

@anaghapamidi we need to add a new section for policies that apply for all cloud providers. The Prisma ID will come later.

gruebel committed:
Update code-security/policy-reference/supply-chain-policies/terraform-policies/ensure-terraform-module-sources-use-git-url-with-commit-hash-revision.adoc

Co-authored-by: Taylor <28880387+tsmithv11@users.noreply.github.com>

tsmithv11 committed:
Update ensure-terraform-module-sources-use-git-url-with-commit-hash-revision.adoc

arane26 committed:
[Compute AG] fixed few typos and spelling (#1455)

jenjoe22 committed:
fix for issue (#1451)

tsmithv11 committed:
Add OIDC check (#1443)

* Update aws-iam-policies.adoc

* Create iam-358.adoc

* Update code-security/policy-reference/aws-policies/aws-iam-policies/aws-iam-policies.adoc

Co-authored-by: Anton Grübel <anton.gruebel@gmail.com>

* Update code-security/policy-reference/aws-policies/aws-iam-policies/iam-358.adoc

Co-authored-by: Anton Grübel <anton.gruebel@gmail.com>

* Update code-security/policy-reference/aws-policies/aws-iam-policies/iam-358.adoc

Co-authored-by: Anton Grübel <anton.gruebel@gmail.com>

* Update code-security/policy-reference/aws-policies/aws-iam-policies/iam-358.adoc

Co-authored-by: Anton Grübel <anton.gruebel@gmail.com>

* Update iam-358.adoc

---------

Co-authored-by: Anton Grübel <anton.gruebel@gmail.com>

georgmarkarian committed:
Update flow-logs-compression.adoc (#1440)

* Update flow-logs-compression.adoc

gcp flow log and the flow log compression doesn't work, if the service user and the storage bucket don't reside on the same project, unless dataflow api is enabled on both

* Update flow-logs-compression.adoc

revised

---------

Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

tsmithv11 committed:
Fix resource (#1439)

tsmithv11 committed:
Update secrets-policy-index (#1435)

bruno-motacardoso committed:
Fix wrong argument on CKV_AZURE_139 policy (#1428)

* Fix wrong argument

* Update ensure-azure-acr-is-set-to-disable-public-networking.adoc

fixed syntax and phrasing

---------

Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

tsmithv11 committed:
Update IAM doc from user input (#1438)

eric-bischoff committed:
Update v30_01_153.adoc (#1431)

Commit:
[Compute AG: App-Embedded Defender]: Reuse content with fragments

Commit:
[Compute AG]: Modularize ACI page with fragments

Commit:
[Compute AG: App-Embedded Defender]: Add GCR page

Pubs-MV committed:
[Compute AG] -Kernel req update for PCSUP-17066 (#1462)

* [Compute AG] -Kernel req update for PCSUP-17066

https://redlock.atlassian.net/browse/PCSUP-17066

* Update system_requirements.adoc

arane26 committed:
[CSPM AG] post-23.8.1 attack path GA updates (#1452)

* [CSPM AG] post-23.8.1 attack path GA updates

* [CSPM AG]  PM feedback

* Update asset-inventory.adoc

* work in progress updates - Ashwini to continue

* [CSPM AG] 23.8.2 attack path updates

---------

Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

Aishbn committed:
[CSPM RN] 23.8.3 - Aug 11th publish (#1458)

* Removed RLP-102046 as it was fixed in 7.2

* Added CIS AWS Foundations Benchmark v2.0.0 in 8.2

* 8.2 Ingestion Updates for GCP pre-reqs

* Initial commits for 8.2 before review.

* Post review updates for 8.2

* url fix

doc1024 committed:
Update prisma-cloud-policies.adoc (#1437)

Pubs-MV committed:
Updates to cost for PCSUP-17733 (#1465)

https://redlock.atlassian.net/browse/CWP-50822

Pubs-MV committed:
Update agentless-scanning.adoc

Pubs-MV committed:
Update look-ahead-planned-updates-prisma-cloud-compute.adoc (#1467)

tsmithv11 committed:
Add Composer (#1466)

tsmithv11 committed:
Update bc-aws-iam-45.adoc (#1321)

tsmithv11 committed:
Update s3 tf fix docs to reflect provider version 4 (#1461)

* Update ensure-that-s3-bucket-has-cross-region-replication-enabled.adoc

* Updated code blocks

anaghapamidi committed:
[AppSec RN] 23.8.2 (#1457)

* [AppSec RN] 23.8.2

* Fix the link for API RN

* Moved LA RN to RN

* Added J's RN

* Modify LA for GA

* Add image

Pubs-MV commented:
hey @anaghapamidi please see @tsmithv11 's comment. 
Also confirm that this content is ready to merge . I see a section for supply chain policies and want to make  sure that it not blanck when we publish.

arane26 committed:
[CSPM AG] rename from code sec to app sec RLP-106732 (#1442)

* [CSPM AG] rename from code sec to app sec RLP-106732

* [CSPM AG] code sec to appl sec url updates

JBakstPaloAlto committed:
[AppSec policy ref] Added CI/CD Risks Policies (#1403)

* Add CI/CD policies

* Add CI/CD Policies

* Add CI/CD Policies

* Added ci/cd policies (formally ci/cd risks from Cider)

* fixing errors

* added ci-cd-pipeline-policies.adoc to folder

* add title to ci cd policies

* Editing the policies in folder to match the yml file

* Sorted yaml book alphabetically according to cicd pipeline repo risk topic

* Correct CI/CD file

* fix note format error

* add space

* added fixes

* added fixes to gl-access-server-unrestricted.adoc

* fix gl-access-server-unrest file

* Try fix for gl-access-server-unrestr

* another attempt to fix the gl-server-unrestr file

* fix typos

* Added index to Azure Repos CI/CD Pipeline Policies

* Added index for Bitbucket CI/CD Pipeline Policies

* Added index for CircleCI CI/CD Pipeline Policies

* Added index for Jenkins CI/CD Pipeline Policies and fixed a typo in the jobs-exe-highpriv file

* Renamed CI/CD Pipeline Policies, including all sub sections (GitHub etc.) to CI/CD Risks Policies

* Added index for GitLab CI/CD Risks Policies

* Added index for GitHub CI/CD Risks Policies

* Added cross platform CI/CD risks policies

* Typos possible cross platform commit

* Fix Cross Platform Folder

* Fixed a link in the GitHub index (weak hash) and a typo in Bitbucket.

---------

Co-authored-by: anaghapamidi <94159219+anaghapamidi@users.noreply.github.com>

Commit:
Update troubleshoot-azure-account-onboarding.adoc (#1454)

* Update troubleshoot-azure-account-onboarding.adoc

Added steps and a screenshot for better understanding.

* Update troubleshoot-azure-account-onboarding.adoc

* Update troubleshoot-azure-account-onboarding.adoc

Proof read the updates and corrected some minor issues.

* adding a new file that @pritesh shared on Slack

---------

Co-authored-by: jenjoe22 <110043374+jenjoe22@users.noreply.github.com>
Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

Pubs-MV committed:
[App Sec Policy Ref] Updated the get-started-code-sec-policies.adoc (#1469)

The title and references to "Code Security"

Pubs-MV committed:
[App Security AG] Updated the book.yml with the correct GH bookdir (#1470)

The book dir was code-security/admin_guide
now is application-security/admin_guide

Pubs-MV committed:
[App Sec Policy Ref] Update book.yml to point to the correct book dir (#1471)

Pubs-MV committed:
[PCCE] -Update deploy_in_fips_mode.adoc PSCUP-17938 (#1472)

Update for https://redlock.atlassian.net/browse/PCSUP-17938

ssugandh committed:
CWP-50215: Correct UI Reference for Container Reports (#1412)

tsmithv11 committed:
Add new verifiers for secrets (#1468)

* Update secrets-policy-index.adoc

* Update secrets-policy-index.adoc

Commit:
[CSPM RN]: Fix typo in Look Ahead for Compute PCEE

elvis19891 committed:
Update twistcli_scan_images.adoc

elvis19891 committed:
Update twistcli_scan_images.adoc

change the command for linux scan with containerd

Commit:
[Newton Compute AG: CWP-29368]: Update RedHat same severity info

elvis19891 committed:
Update cis_benchmarks.adoc

Commit:
[Compute AG: CWP-47467]: Support registry tags in Collections

Commit:
Update prisma_cloud_vulnerability_feed.adoc

Commit:
remove duplicate https from the URL

Commit:
Update prisma_cloud_vulnerability_feed.adoc

Commit:
[App-Embedded Defender GCR]: Update gcloud commands

Commit:
Update `twistcli` flags

Commit:
Update `twistcli` flag syntax

Commit:
[Compute AG]: Update redploy Defenders commands

Commit:
Add consistent markup for key:value pairs

Pubs-MV committed:
[CSPM RN for App Sec]Revert "[AppSec RN] 23.8.2" - because GA moved to Aug 28 (#1481)

* Revert "[AppSec RN] 23.8.2 (#1457)"

This reverts commit 100974293e293fe3111069d3c536bd5669943130.

* Update look-ahead-planned-updates-prisma-cloud-code-security.adoc

Pubs-MV committed:
Delete cspm/admin-guide/connect-your-cloud-platform-to-prisma-cloud/onboard-your-aws-account directory

This is replaced by /onboard-aws

Pubs-MV committed:
Delete cspm/admin-guide/connect-your-cloud-platform-to-prisma-cloud/onboard-your-gcp-account directory (#1482)

this is replaced  by onboard-gcp

Commit:
Fix typo

Commit:
Fix inconsistent note

rcaballeromx committed:
[Compute AG] Add link to EBS encryption permission list (#1479)

Co-authored-by: “rcaballeromx” <“rcaballero@paloaltonetworks.com”>

rcaballeromx committed:
[Compute AG]Add curl command flag for CA (#1478)

Co-authored-by: “rcaballeromx” <“rcaballero@paloaltonetworks.com”>

rcaballeromx committed:
[Newton] [Compute RN] PCEE (#1446)

* [Newton] [Compute RN] PCEE

* Add to book.yml

* API change for CWP-50127(CWP-41710)

* Update build number

* Update build number

* [Compute PCEE RN]: Fix typos and add image

* fixed  the description for deprecation extension for CNNS and added bug IDs for blurbs

---------

Co-authored-by: “rcaballeromx” <“rcaballero@paloaltonetworks.com”>
Co-authored-by: Saurabh Sugandh <ssugandh@paloaltonetworks.com>
Co-authored-by: divyabhushan <divya_bhushan@hotmail.com>
Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

rcaballeromx committed:
[Compute RN] [Newton] Remove Look Ahead information (#1448)

* [Compute RN] [Newton] Remove Look Ahead information

* Update look-ahead-planned-updates-prisma-cloud-compute.adoc

added .129 to replace .xxx

---------

Co-authored-by: “rcaballeromx” <“rcaballero@paloaltonetworks.com”>
Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

Commit:
Add non-rpm image to cspm/rn/_graphics (#1484)

Commit:
[Compute AG]: Include anchors for TOC (#1486)

rcaballeromx committed:
[Compute AG]Newton System Requirements (#1433)

Co-authored-by: “rcaballeromx” <“rcaballero@paloaltonetworks.com”>

mikeurbanski1 committed:
Update code-security-licensing-configuration.adoc (#1487)

* Update code-security-licensing-configuration.adoc

* Update code-security-licensing-configuration.adoc

commented out the credits and fixed some grammar.

---------

Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

rcaballeromx committed:
[Compute RN] [Newton] PCCE (#1447)

* [Compute RN] [Newton] PCCE

* Added API change for CWP-50127 (CWP-41710)

* Add release details

* Update build number

* [Newton RN]: Typos fix

* Fix version number and Add image for non-rpm cve details

* Update release-notes-31-00.adoc

fixed the blurbs for deprecation for CNNS and Code Repo scanning and added a new section

* Update known-issues-31-00.adoc

revised the formatting to match our style.

* Update release-notes-31-00.adoc

added the fixed issues here.

For PCCE RN, the fixes are added to the version

* Update known-issues-31-00.adoc

* Update book.yml

commented out the known issues  file as it was empty.

* Add CWP references

---------

Co-authored-by: “rcaballeromx” <“rcaballero@paloaltonetworks.com”>
Co-authored-by: Saurabh Sugandh <ssugandh@paloaltonetworks.com>
Co-authored-by: divyabhushan <divya_bhushan@hotmail.com>
Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

Pubs-MV committed:
[Compute AG]Update book_compute_edition.yml for Newton (31) (#1488)

* Update book_compute_edition.yml for Newton (31)

* Update book_compute_edition.yml

Commit:
[Compute LookAhead]: Clear messaging of the release (#1489)

* [Compute LookAhead]: Clear messaging of the release

* Update look-ahead-planned-updates-prisma-cloud-compute.adoc

thank you for the edit @divyabhushan.

Commenting it out now as we will republish this week for a different release and we can remove this notice for Aug 20 that has occurred.

---------

Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

Commit:
[Compute AG: CWP-38685]: Add a note on Docker registry v1 deprecation (#1490)

* [Compute AG: CWP-38685]: Add a note on Docker registry v1 deprecation

* Update configure_registry_scanning.adoc

removd please

---------

Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

arane26 committed:
[CSPM RN] post-23.8.2 fixes (#1495)

Pubs-MV committed:
[App Sec GA/CSPM RN] (#1492)

* [App Sec GA/CSPM RN]
merged the changes that @anaghpamidi had in #1457 that I had reverted in #1481.

Then moved and revised stuff that was just lying around.

* Add JIRA tickets

* Update look-ahead-planned-updates-prisma-cloud-code-security.adoc

Updates for RLP-111206.  @manukumar6

---------

Co-authored-by: anaghapamidi <94159219+anaghapamidi@users.noreply.github.com>

Pubs-MV committed:
Look ahead for Newton Update 1 31.01.xx (#1500)

rcaballeromx committed:
[Compute AG][CWP-47465][Newton]Deprecated PodSecurityPolicy note (#1497)

Co-authored-by: “rcaballeromx” <“rcaballero@paloaltonetworks.com”>

Pubs-MV committed:
Update drift-detection.adoc (#1503)

worked in Taylor's changes that were in PR-1410;

These were somehow rolled back @anaghapamidi

Pubs-MV committed:
[CSPM AG]Update for RLP-89822 (#1504)

* [CSPM AG]Update for RLP-89822

* Update view-respond-to-prisma-cloud-alerts.adoc

Commit:
[CSPM RN]Compute PCEE - Look ahead for Newton Update 1 (31.01.xx) (#1501)

* [NewtonUpdate1 - LookAhead]: Fix spacing

* Update release number for the line to be published on Sep 10

* Update look-ahead-planned-updates-prisma-cloud-compute.adoc

removed APIs in EoS notice based on Tal's comment.

* Update look-ahead-planned-updates-prisma-cloud-compute.adoc

added more info on the APIs supported from Elichy

---------

Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

Commit:
Delete extra prisma-cloud-docs dir

Pubs-MV committed:
Update vm_image_scanning.adoc (#1510)

updates for https://redlock.atlassian.net/browse/PCSUP-18285

Pubs-MV committed:
Update view-respond-to-prisma-cloud-alerts.adoc (#1520)

added pending resolution info for https://redlock.atlassian.net/browse/RLP-112184

Commit:
[CSPM Alerts Fix]: Update typo in screenshot (#1507)

* [CSPM Alerts Fix]: Update typo in screenshot

* [CSPM Alerts]: General grammar/language updates

* Update send-prisma-cloud-alert-notifications-to-third-party-tools.adoc

---------

Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

jjchavanne committed:
removed duplicates, made corrections, & improvements (#1521)

Co-authored-by: John Chavanne <jchavanne@paloaltonetworks.com>

eric-bischoff committed:
DISA STIG Release Findings TechDoc Structure Update  (#1505)

* Create v30_01_153.adoc

DISA STIG release findings for Maxwell Update 1 (30.01.153)

* Update v30_01_153.adoc

* Create v30_02_123.adoc

DISA STIG Release Findings for Maxwell Update 2 Console & Defender Images (30.02.123)

* Update v30_02_123.adoc

* Update for DISA STIG Release Findings

* Update book.yml to add 30.02.123 file

* Update v30_02_123.adoc with correct link

* Update v30_01_153.adoc

fixed the formatting errors and Gsheet link

* Update releases.adoc

* DISA STIG Release Findings TechDoc Structure Update 2

DISA STIG Release Findings TechDoc Structure Update 2

---------

Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

jenjoe22 committed:
IAM Query Conditions update  (#1508)

* first draft

* Update iam-query-conditions.adoc - format fix

---------

Co-authored-by: Aishwarya <84237320+Aishbn@users.noreply.github.com>

Commit:
Fix tables in runtime_audits.adoc (#1253)

* Fix tables in runtime_audits.adoc

Some cells are missing, causing mismatch in the tables : 
 - In 'Runtime detections for processes' table, two cells missing for 'Port Scanning' row : 'DETECTION' and 'CONTEXT' columns
 - In 'Runtime detections for Network activities' table, three cells missing in 'Port Scanning' row : 'AUDIT MESSAGE', 'TRIGGER AN INCIDENT', and 'WORKLOADS' columns

* [Compute AG]: Update Runtime Audits missing data

---------

Co-authored-by: divyabhushan <divya_bhushan@hotmail.com>

Commit:
[Compute AG: CWP-42722]: certificate info for webhook (#1476)

* [Compute AG: CWP-42722]: certificate info for webhook

* Add conjunction and fix apostrophes

* Revise the sentense about webhook support

* Update Webhooks section uniquely for SaaS and On-prem

* Update webhook section in Docker Hub with image and Nexus registry

* Update JFrog trusted certificate info for webhooks

---------

Co-authored-by: “rcaballeromx” <“rcaballero@paloaltonetworks.com”>

JBakstPaloAlto committed:
[AppSec AG]Verify Integration for Code Repositories, Quick Start (#1491)

* Added integration verification for code repositories

* Modified Code Repositories intro (slightly)

* Added graphic verification to code repos

* Added Quick Start Guide

* Fix links

* Minor typos

* Rename quick-start-guide.adoc to quick-start.adoc

* Update book.yml

* Update quick-start.adoc

---------

Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

Pubs-MV committed:
fixed broken book (#1522)

anaghapamidi committed:
[AppSec RN] 23.9.1 (#1517)

* Add LA for 9.1

* Update  in  Security

* Format fix for LA dep notice

---------

Co-authored-by: Aishwarya <84237320+Aishbn@users.noreply.github.com>

Aishbn committed:
[CSPM RN] Sept 8th Publish (#1511)

* Includes 9.1 GCP pre-req permissions update

* Added September 2023 to the list

* Initial commit for Features Inctroduced 23.9.1

* LA -Initial 9.1 commit

* LA- added IAM policy update for 9.1

* Includes policy updates and ingestion re-order.

* Moved RLP-109870 from LA

* Post review updates for 9.1

* removed deprecation notice

* changed sunset release date to 10.2 in LA

* Added CAS policies

* Update features-introduced-in-september-2023.adoc

updated the blurb for RLP-112184 to remove the Impact reference and higher alert count based on PM feedback

---------

Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

Commit:
[CSPM Newton Update 1]: Release Notes for 31.01.xxx PCEE (#1518)

* [CSPM Newton Update 1 RN]

* Update LookAhead

* Update features-introduced-in-compute-september-2023.adoc

added  123 for .xxx

* Update features-introduced-in-compute-september-2023.adoc

added the API info

* Update features-introduced-in-compute-september-2023.adoc

fixed the heading

* Update look-ahead-planned-updates-prisma-cloud-compute.adoc

* Update prisma-cloud-compute-release-information.adoc

---------

Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

iansk committed:
[CWP Public Sector] Fix book so it uploads to AEM (#1523)

Commit:
[Newton Update 1 - Compute RN PCEE]: Few fixes (#1524)

* [Compute RN PCEE]: Remove placeholder headers

* Update version number

* Fix spacing between the `tt` tag and the heading

* Screenshot update in the latest release

Commit:
[Compute AG]: Newton update 1 integration branch (#1526)

* [Compute AG: Newton]: CWP-37603 - Deprecate Docker Access Control (#1419)

* [Newton Update1 AG: CWP-37603]: Deprecate Docker access control and Access User role

* Fix broken xrefs

* Revert twistcli_scan_images.adoc

--docker-tlscert flag is needed

* [Compute AG: CWP-51586]: Update backup and restore doc (#1516)

* [Compute AG: CWP-51586]: Update backup and restore doc

* Update disaster_recovery.adoc

---------

Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

* [Compute AG: CWP-42985]: Support package type scope for vulnerability policy (#1499)

* [Compute AG: CWP-42985]: Support per-package severity

* [Compute AG: NewtonUpdate1]: Improve vuln mgmt rules page

* [Compute AG]: Update blocking on vuln severity section

* [CWP-49316 and 48931]: Support Parallel Registry Scan  (#1494)

* [CWP-49316]: Env variable to edit parallel registry scan limit

* [Compute AG: CWP-49316]: Update parallel registry scan info

* New registry in queue until Defender is free

* Update "Add and Scan" labeling

---------

Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

anaghapamidi committed:
[AppSec RN] Add REST API RN (#1525)

* Add LA for 9.1

* Update  in  Security

* [AppSec RN] Add Rest API RN

* Change the Fix version in LA

* Update look-ahead-planned-updates-prisma-cloud-code-security.adoc

replaced sunset version from 9.1 to 9.2

* Fix LA

* Fix format

---------

Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

kade2711 committed:
Update onboard-aws.adoc (#1531)

* Update onboard-aws.adoc

REMOVED EXTRA WORDS | @Pubs-MV need to change link preview on line 9 to show "Aws" - Currently, it shows "GCP" instead, but we are on the AWS page. It does link correctly, however.

* Update onboard-aws.adoc

---------

Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

Commit:
[PCEE Compute RN: Newton Update 1]: Add fixed issue for CWP-50733 (#1535)

* Add fixed issue for CWP-50733

* Update prisma-cloud-compute-known-issues.adoc

formatting for fixed issues in PCEE.

removed //CWP-45194 - this will be fixed in 22.12.xxx Lagrange Update 3
|*-*  
|The https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admincompute/waas/waas_api_discovery#_inspect_discovered_endpoints[API endpoints discovered] on App-embedded deployments are missing workload values and show zero vulnerabilities, although the protected workload has vulnerabilities.

as it was fixed in Lagrange update 3 (which is too old to keep here)

---------

Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

Pubs-MV committed:
Update releases.adoc (#1533)

* Update releases.adoc

* Update releases.adoc

Pubs-MV committed:
[App Sec AG] Updates for Secrets Security language (#1532)

Commit:
Adding files to rebase
[Compute AG]: Remove rbac.adoc file and its references (#1529)

* Remove rbac.adoc file and its references

* [Compute AG]: Fix syntax

tsmithv11 committed:
Add docs for Azure policies (#1512)

* Update azure-general-policies.adoc

* Create azr-general-85.adoc

* Update azure-networking-policies.adoc

* Create azr-networking-63.adoc

* Create azr-networking-64.adoc

* Update azr-networking-64.adoc

* Update azure-networking-policies.adoc

* Update azure-networking-policies.adoc

* Update azr-networking-64.adoc

* Update azr-networking-63.adoc

* Update azr-general-85.adoc

* Update azr-networking-63.adoc

* Update azr-networking-64.adoc

* Modified book.yml

---------

Co-authored-by: anaghapamidi <94159219+anaghapamidi@users.noreply.github.com>

Commit:
[Compute AG: CWP-49173]: Add traffic sampling info (#1527)

rcaballeromx committed:
[Compute][PCCE RN] Add release notes for 30.01 (#1509)

* [Compute][PCCE RN] Add release notes for 30.01

* Remove Secret Scanning

* [Compute RN]: Add blurb for CWP-35771

* [Compute RN]: Add blurb for CWP-42985

* Remove RN blurb for CWP-40710 - moved to Update 2

* Add blurb for CWP-48931

* Add deprecation notice for CWP-37603

* Add known issue for v30.xx

* Add addressed issue for CWP-49173

* Update release-notes-31-01-update1.adoc

added build no and date

* Fix end of support notice

* Update version number

* [Compute RN: Newton Update 1]: Add addressed issue for CWP-50733

* [Newton RN]: Add known issue for CWP-50733

* Update release-notes-31-01-update1.adoc

Remove the workaround for older versions for CWP-49173

* Remove blurb for CWP-50923 - this is still a Bug

* Add known issue for CWP-50923 - AKS cluster

* Add SHA and Update path

* Add screenshots

* Update release-notes-31-00.adoc

fixed the version nos. that were incorrect

---------

Co-authored-by: “rcaballeromx” <“rcaballero@paloaltonetworks.com”>
Co-authored-by: divyabhushan <divya_bhushan@hotmail.com>
Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

Pubs-MV committed:
PCCE RN - Update book.yml (#1537)

Fixed the broken link to 

ERROR:root:Topic file doesn't exist: release-information/known-issues-31-00.adoc

Commit:
[Compute PCCE RN: Correction] Remove blurb for CWP-50923 - this is still a Bug (#1538)

* Remove blurb for CWP-50923 - this is still a Bug

* Remove workaround for Lagrange and Kepler release - not relevant

* [Compute RN Newton]: Also remove blurb for CWP-50923 from 31.00 RN

Commit:
[Compute RN PCEE]: CWP-50923: Still a bug and not fixed (#1536)

* CWP-50923: Still a bug and not fixed

* Update prisma-cloud-compute-known-issues.adoc

@divyabhushan fixed the formatting. We do not use a heading

---------

Co-authored-by: Pubs-MV <mvaidyanathan@paloaltonetworks.com>

Pubs-MV committed:
[CSPM AG] Audit log forwarding duplication (#1543)

* Update define-prisma-cloud-enterprise-settings.adoc

* Update view-audit-logs.adoc to consolidate information

* updated title

Pubs-MV committed:
fixed xref (#1544)

Commit:
[Compute PCCE RN]: Update code name to 31.01 (#1540)

* [Compute PCCE RN]: Update code name to 31.01

* [CWP-49173]: Update on traffic sampling in AG and RN

* Feedback from the Slack message

Pubs-MV committed:
Update prisma-cloud-compute-release-information.adoc (#1545)

manukumar6 committed:
RLP-115284 (#1546)

manukumar6 committed:
Compute API v31.01 fix (#1551)

* RLP-115284

* RLP-115284

* RLP-115284

* RLP-115284

arane26 committed:
[CSPM AG] content fixes (#1547)

* [CSPM AG] content fixes

* Update troubleshoot-aws-errors.adoc

Syntax fixes for the first task.

* Update prisma-cloud-integrations.adoc

Added a note based on RLP-91930.

---------

Co-authored-by: Aishwarya <84237320+Aishbn@users.noreply.github.com>

jjchavanne committed:
fix syntax and expalantions (#1530)

Co-authored-by: John Chavanne <jchavanne@paloaltonetworks.com>

Aishbn committed:
[CSPM RN] 23.9.2 September 22nd Publish (#1548)

* Initial commit for 23.9.2 - before review.

* [AppSec] Add Sholmi's Feedback

* Post review changes

* [AppSec] Update the RN with latest suggestion

* Title update for CAS Deprecation Notice

* Azure API ingestion update - Last min requests

* Moved RLP-114884 from LA to current release. Last min req.

---------

Co-authored-by: anaghapamidi <94159219+anaghapamidi@users.noreply.github.com>

Pubs-MV committed:
Update features-introduced-in-september-2023.adoc (#1554)

added blurb for https://redlock.atlassian.net/browse/RLP-116013

anaghapamidi committed:
Resolve conflicts and merge with origin/master

anaghapamidi committed:
Rebase to origin/master

anaghapamidi committed:
Merge changes to match origin/master

anaghapamidi committed:
Resolve conflicts

anaghapamidi reviewed (commented):
Have opened a [new PR ](https://github.com/PaloAltoNetworks/prisma-cloud-docs/pull/1559)to merge these changes with the GA changes. 

cc: @Pubs-MV
